src/main/java/org/dependencytrack/model/IntegrityAnalysisComponent.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+package org.dependencytrack.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import javax.jdo.annotations.Column;
+import javax.jdo.annotations.IdGeneratorStrategy;
+import javax.jdo.annotations.PersistenceCapable;
+import javax.jdo.annotations.Persistent;
+import javax.jdo.annotations.PrimaryKey;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+/**
+ * Tracks integrity analysis results for components tracked by their uuid's.
+ *
+ * @author Meha Bhargava
+ */
+@PersistenceCapable(table = "INTEGRITY_ANALYSIS_COMPONENT")
```

Comment:
Should we call it INTEGRITY_ANALYSIS or COMPONENT_INTEGRITY_ANALYSIS ?
It might be more readable and aligned with 'ComponentIntegrityQueryManager'